contracts/ExitFormat.sol

```diff
@@ -81,8 +81,16 @@ library ExitFormat {
         for (uint256 i = 0; i < exit.length; i++) {
             address asset = exit[i].asset;
             for (uint256 j = 0; j < exit[i].allocations.length; j++) {
+                require(
+                    _isAddress(exit[i].allocations[j].destination),
+                    "Destination is not an address"
```

Comment:
Can we say "is not a padded address" or similar?

Reply:
We can!